### PR TITLE
Basic redux docs

### PR DESCRIPTION
**Pull request by tmeasday**
I didn't include a lot of the thunk content from http://docs.apollostack.com/apollo-client/redux.html because it doesn't seem like you'd use it w/ `react-apollo`. Perhaps we want to keep it for the "raw" Client docs? I'm not sure.

### CODE REVIEW

**Comment by stubailo** [comment]
Can we add this as well? https://github.com/apollostack/docs/pull/179

File: source/redux.md [hunk 1]
```diff
@@ -3,14 +3,12 @@ title: Integrating with Redux
 order: 15
 ---
 
-XXX: do we want to include James' stuff about MobX + React router?
 
-Explain how to use custom reducers for Apollo Client-related data
+By default, the Apollo Client creates its own internal Redux store to manage queries and their results. If you are already using Redux for the rest of your app, you can have the client integrate with your existing store instead. This will let you better track the different events that happen in your app, and how your client and server side data changes interleave.
```
Comment (helfer) [inline source/redux.md]: I feel like this is advice that you should have just one Redux store for your whole app, but it doesn't really go into why. Can we link to some resource that explains it?

**Comment by helfer** [comment]
Looks fine to me so far. 👍

File: source/redux.md (hunk 1 above)
Comment (stubailo) [inline source/redux.md]: I think the main benefit is looking at your dev tools, and seeing one stream of events - also, if you're doing SSR/store hydration, you just have to send one blob to the client.

**Comment by tmeasday** [comment]
> so far

Does that mean you think there should be more here? ;)

**Comment by helfer** [comment]
Yep, I meant the dev tools stuff 😉

**Comment by helfer** [comment]
Merged!

**Pull request merged**